gnmi_server/client_subscribe.go

```diff
@@ -130,6 +139,11 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 	if err != nil {
 		return grpc.Errorf(codes.NotFound, "Invalid subscription path: %v %q", err, query)
 	}
+
+	if connectionKey, valid = validateConnectionRequest(c.addr, c.threshold, query.String()); !valid {
+		return grpc.Errorf(codes.ResourceExhausted, "Server connections are at capacity.")
```

Comment:
I believe that the UNAVAILABLE code makes more sense here to indicate a transient error.

Reply:
Agree, will change.

Reply:
Done